docs/comparisons/opa.md

```diff
@@ -0,0 +1,204 @@
+
+## CI/CD integration
+
+Both OPA Gatekeeper and Kubewarden can be managed using GitOps methodologies.
+
+However, in the context of OPA Gatekeeper, there's a coupling between the policy's source code
+(the Rego code) and the Custom Resource used to deploy it inside of Kubernetes.
+This introduce some extra steps inside of CI/CD pipelines.
+
+Rego has [testing tools](https://www.openpolicyagent.org/docs/latest/policy-testing/)
+that allow the creation of unit tests suites. Writing tests and executing them inside of
+a CI/CD pipeline is essential to ensure policies behave as expected.
+
+To use these testing tools, the source code of the policy must be made available
+inside of dedicated text files. It's not possible to read the source code from the YAML
+files used to declare the OPA Gatekeeper policy.
+The CI/CD pipeline must keep in sync the Rego source code being tested with the code
+defined inside of the OPA Gatekeeper Custom Resource. This can be done using some 3rd
+party tools.
+
+Kubewarden policies have CI/CD pipelines like the ones of traditional microservices.
+Usually their source code lives inside of a Git repository and then, using
+traditional CI/CD systems, unit tests are ran against it. The unit tests are
+written using the testing frameworks of the language used to write the policy.
+Once all the tests are passing the policy is compiled to WebAssembly and pushed
+to a container registry.
+This kind of pipeline is usually maintained by the policy author.
+
+Kubernetes administrators typically maintain other automation pipelines that react to
+new releases of the policy (leveraging automation tools like
+[Dependabot](https://docs.github.com/en/code-security/dependabot/working-with-dependabot),
+[Renovate bot](https://www.mend.io/renovate/),
+[updatecli](https://www.updatecli.io/) and others) or to changes to the
+policy configuration.
+
+The pipeline tests the policy against different types of requests. The testing can be done using
+the [kwctl](https://github.com/kubewarden/kwctl) cli tool, without requiring a running
+Kubernetes cluster.
+
+## Policy enforcement modes
+
+Both OPA Gatekeeper and Kubewarden can deploy policies using two different operation mode:
+
+- `deny`: violation of a policy causes the request to be rejected
+- `warn`: violation of a policy does not cause the request to be rejected. The
+  violation is just logged for auditing purposes
+
+## Deployment mode
+
+All the OPA Gatekeeper policies are evaluated by the same server.
+On the other hand, Kubewarden allows multiple evaluation servers to be defined.
+These servers are defined by a Custom Resource called `PolicyServer`.
+
+When declaring a Kubewarden policy, the Kubernetes administrator can then decide
+which `PolicyServer` is going to host it.
+
+:::note
+The `PolicyServer` object is just a high level abstraction introduced by Kubewarden.
+Behind the scenes a Deployment with a specific replica size is created.
```

Comment:
```suggestion
The `PolicyServer` object is a high level abstraction introduced by Kubewarden.
Behind the scenes a `Deployment` with a specific replica size is created.
```

I'm not quite sure about whether to quote or decapitalize 'deployment'. Quotes probably good.